charts/kong/example-values/minimal-kong-controller-konnect.yaml

```diff
@@ -0,0 +1,21 @@
+# Basic values.yaml configuration for Kong for Kubernetes with the ingress controller
+# and the controller's Konnect integration enabled.
+
+image:
+  repository: kong
+  tag: "3.1"
+
+env:
+  database: "off"
+
+ingressController:
+  enabled: true
+
+  image:
+    repository: kong/nightly-ingress-controller
+    tag: "2023-02-15"
+    effectiveSemver: "2.9.0"
+
+  konnect:
+    enabled: true
+    runtimeGroupID: "523f592f-9828-48a2-ab2e-cf113276369d"
```

Comment:
There's no generic value possible here, correct? We probably need some obviously invalid all zero UUID with a CHANGEME a la https://github.com/Kong/charts/blob/2f73810d66e5ad7c1cebb33b74887f7b8ba7db32/charts/kong/example-values/minimal-kong-enterprise-dbless.yaml#L12 I think.